tests/specialization_constants/specialization_constants_exceptions_catching_common.h

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Common expected exceptions catching for specialization constants
```

Comment:
It is still unclear to guess what you are checking here from this single line

Reply:
i've extend this comment in this commit: ba22e08858bf498a0449b391dd9681522ffa4a85